cmd/celvet/main.go

```diff
@@ -81,23 +92,58 @@ func main() {
 	}
 
 	limitErrors := celvet.CheckMaxLimits(structural)
+	costErrors, compileErrors, otherErrors := celvet.CheckExprCost(structural)
+	if useJSON {
+		emitJSON(limitErrors, costErrors, compileErrors, otherErrors)
+	} else {
+		emitText(limitErrors, costErrors, compileErrors, otherErrors, *humanReadable)
+	}
+
+	if len(limitErrors)+len(costErrors)+len(compileErrors) > 0 {
+		os.Exit(1)
+	}
+}
+
+func emitJSON(limitErrors []error, costErrors []*celvet.CostError, compileErrors []*celvet.CompilationError, otherErrors []error) {
+	type JSONOutput struct {
```

Comment:
Mark all fields that are optional and lists that can be empty with `omitempty` to make json output more compact/readable?

Reply:
Gotcha, fixed